src/prisma/schema.prisma

```diff
@@ -4067,3 +4069,19 @@ model BlockchainLog {
   @@index([contractId])
   @@index([eventName])
 }
+
+model GithubIssue {
+  id            String      @id @default(uuid()) @db.Uuid
```

Comment:
we shouldn't really need our own id, i'd avoid adding this column and instead ad a unique index on issueNumber and repoId?